test/test-errors.js

```diff
@@ -251,3 +251,28 @@ test('errors for Not-of-<PExpr>', function(t) {
 
   t.end();
 });
+
+test('Error where left-right associativity is unclear', function(t) {
+  try {
+    ohm.grammar(
+      'G {\n' +
+      '  A = A "+" A -- add \n' +
+      '  | B\n' +
+      '  B = "117"\n' +
+      '}');
+    t.fail('Expected an exception to be thrown');
```

Comment:
The more idiomatic way to ensure an exception is thrown is to use `t.throws()`: https://github.com/substack/tape#tthrowsfn-expected-msg